scripts/postinstall.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node

```

Comment:
@linusklingzell Is the she-bang needed when we're running this with `node scripts/postinstall.js`?